tests/models/test_dag.py

```diff
@@ -3825,16 +3836,67 @@ def test_clear_upstream_not_your_setup(self):
             s1, w1, w2, t1 = self.make_tasks(dag, "s1, w1, w2, t1")
             s1 >> w1 >> t1
             s1 >> w2
-            self.cleared_upstream(w2) == {s1, w2, t1}
+            # w2 is downstream of s1, so when clearing upstream, it should clear s1 (since it
+            # is upstream of w2) and t1 since it's the teardown for s1 even though not downstream of w1
+            assert self.cleared_upstream(w2) == {s1, w2, t1}
 
-    def clearing_teardown_no_clear_setup(self):
+    def test_clearing_teardown_no_clear_setup(self):
         with DAG(dag_id="test_dag", start_date=pendulum.now()) as dag:
             s1, w1, t1 = self.make_tasks(dag, "s1, w1, t1")
             s1 >> t1
             # clearing t1 does not clear s1
-            self.cleared_downstream(t1) == {t1}
+            assert self.cleared_downstream(t1) == {t1}
             s1 >> w1 >> t1
             # that isn't changed with the introduction of w1
-            self.cleared_downstream(t1) == {t1}
+            assert self.cleared_downstream(t1) == {t1}
             # though, of course, clearing w1 clears them all
-            self.cleared_downstream(w1) == {s1, w1, t1}
```

Comment:
Did you intend to delete this, instead of just adding the missing assert?

Reply:
lemme confirm